### PR TITLE
Issue-144: Add unique page titles.

### PR DESCRIPTION
**Pull request by cindyli**
* [X] This pull request has been linted by running `npm run lint` without errors
* [X] This pull request has been tested by running `npm run dev` and reviewing affected routes
* [X] This pull request has been built by running `npm run generate` without errors
* [X] This isn't a duplicate of an existing pull request

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/144.

Add unique titles for each page.

## Steps to test

1. Visit each page;
2. Make sure they all have unique titles in the format of "{title} | We Count".

### CODE REVIEW

File: pages/news/page/_num/index.vue [hunk 1]
```diff
@@ -17,6 +17,11 @@ export default {
 			currentPageNum: parseInt(this.$route.params.num)
 		}
 	},
+	head () {
+		return {
+			titleTemplate: this.title + " | %s"
```
Comment (greatislander) [inline pages/news/page/_num/index.vue:22]: I'd include the page number, so:

'News (Page %d) | %s'

Comment (greatislander) [reply pages/news/page/_num/index.vue:22]: (Same for all other paginated pages.)

**Comment by cindyli** [comment]
Comments addressed and ready for another look.

File: pages/_slug/index.vue [hunk 2]
```diff
@@ -16,6 +16,11 @@ export default {
 			content: ""
 		}
 	},
+	head () {
+		return {
+			titleTemplate: this.title + " | %s"
```
Comment (TeddTech) [inline pages/_slug/index.vue:21]: Out of curiosity. How does `%s` know to render as "We Count"

Comment (greatislander) [reply pages/_slug/index.vue:21]: https://vue-meta.nuxtjs.org/api/#titletemplate

**Pull request merged**